makefiles/boot/riotboot.mk

```
@@ -79,7 +79,7 @@ riotboot/flash-bootloader: riotboot/bootloader/flash
 riotboot/bootloader/%:
 	$(Q)/usr/bin/env -i \
 		QUIET=$(QUIET) PATH=$(PATH)\
-		BOARDSDIR=$(BOARDSDIR) BOARD=$(BOARD)\
+		EXTERNAL_BOARD_DIRS=$(EXTERNAL_BOARD_DIRS) BOARD=$(BOARD)\
```

Should be:
```suggestion
		EXTERNAL_BOARD_DIRS="$(EXTERNAL_BOARD_DIRS)" BOARD=$(BOARD)\
```

Otherwise I get the following:

```
EXTERNAL_BOARD_DIRS=../../../external-boards make BOARD=nucleo-l433rc -C tests/riotboot -j3
make: Entering directory '/home/francisco/workspace/RIOT2/tests/riotboot'
Building application "tests_riotboot" for "nucleo-l433rc" with MCU "stm32l4".

/usr/bin/env: ‘/home/francisco/workspace/external-boards’: Permission denied
/home/francisco/workspace/RIOT2/makefiles/boot/riotboot.mk:80: recipe for target 'riotboot/bootloader/binfile' failed
make: *** [riotboot/bootloader/binfile] Error 126
make: *** Waiting for unfinished jobs....
make: *** wait: No child processes.  Stop.
```

---

Nice catch. Fixed and squashed right in. (Clicking on "force-pushed" should show the diff.)